README-bn.md

```diff
@@ -0,0 +1,197 @@
+# কুবারনেটিস ডকুমেন্টেশন
+
+[![Build Status](https://api.travis-ci.org/kubernetes/website.svg?branch=master)](https://travis-ci.org/kubernetes/website)
+[![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
+
+স্বাগত! এই সংগ্রহস্থলে [Kubernetes ওয়েবসাইট এবং ডকুমেন্টেশন](https://kubernetes.io/) তৈরি করার জন্য প্রয়োজনীয় সমস্ত উপাদান রয়েছে। আমরা খুবই আনন্দিত যে আপনি অবদান রাখতে চান!
+
+
+- [ডকুমেন্টেশন এ অবদান](#contributing-to-the-docs)
+- [স্থানীয়করণ ReadMeগুলি](#localization-readmemds)
+
+## `README.md` এর স্থানীয়করণ কুবারনেটস ডকুমেন্টেশন
+আপনি বাংলা স্থানীয়করণের রক্ষণাবেক্ষণকারীদের (maintainers) কাছে পৌঁছাতে পারেন:
+* MD Shahriyar Al Mustakim Mitul ([LinkedIn](https://www.linkedin.com/in/md-shahriyar-al-mustakim-mitul-9084b31a0/), [Twitter](https://twitter.com/ShahriyarMitul), [GitHub](https://github.com/mitul3737))
+* MD Ikramul Kayes ([LinkedIn](https://www.linkedin.com/in/md-ikramul-kayes-753674214/), [Twitter](https://twitter.com/IkramulKayes), [GitHub](https://github.com/ikramulkayes))
+* Umme Abira Azmary ([LinkedIn](https://www.linkedin.com/in/umme-abira-azmary-68404a1bb/), [Twitter](https://twitter.com/AbiraAzmary), [GitHub](https://github.com/Mouly22))
+* [Slack channel](https://kubernetes.slack.com/messages/kubernetes-docs-bn)
+
+
+
+## এই সংগ্রহস্থল(repository) ব্যবহার করে
+
+
+আপনি Hugo (বর্ধিত সংস্করণ) ব্যবহার করে স্থানীয়ভাবে ওয়েবসাইটটি চালাতে পারেন, অথবা আপনি এটি একটি কন্টেইনার রানটাইমে চালাতে পারেন। আমরা দৃঢ়ভাবে কন্টেইনার রানটাইম ব্যবহার করার পরামর্শ দিই, কারণ এটি লাইভ ওয়েবসাইটের সাথে স্থাপনার ধারাবাহিকতা দেয়।
+
+## পূর্বশর্ত
+
+এই সংগ্রহস্থলটি(repository) ব্যবহার করার জন্য, আপনাকে লোকাল সিস্টেম বা, ডিভাইস এ নিম্নলিখিত জিনিস ইনস্টল করতে হবে:
+
+- [npm](https://www.npmjs.com/)
+- [Go](https://golang.org/)
+- [Hugo (বর্ধিত সংস্করণ)](https://gohugo.io/)
+- একটি ধারক রানটাইম, উদাহরন হিসেবে [Docker](https://www.docker.com/).
+
+আপনি কাজ শুরু করার আগে, দরকারি জিনিশগুলি ইনস্টল করুন। সংগ্রহস্থল ক্লোন(clone) করুন এবং ডিরেক্টরিতে(directory) প্রবেশ করুন:
+```bash
+git clone https://github.com/kubernetes/website.git
+cd website
+```
+
+কুবারনেটিস ওয়েবসাইটটি [Docsy Hugo থিম](https://github.com/google/docsy#readme) ব্যবহার করে। এমনকি যদি আপনি একটি কনটেইনার ওয়েবসাইট চালানোর পরিকল্পনা করেন, আমরা দৃঢ়ভাবে নিম্নলিখিতগুলি চালিয়ে সাবমডিউল এবং অন্যান্য প্রয়োজনীয় জিনিসগুলিশগুলি  পুল(pull) করার পরামর্শ দিই:
+
+
+```bash
+# pull in the Docsy submodule
```

Comment:
we can localize comments in code blocks like this one if they make sense after localizing

Reply:
okay! So what needs to be done next? What do you suggest?